unitTests/misc/EnvironmentTest.ts

```diff
@@ -5,6 +5,8 @@ declare const _;
 export function EnvironmentTest() {
   describe('Environment', () => {
     it(`the '_' global variable should hold the underscore library`, () => {
+      // If this test fails, it might be because the global variable is being
+      // overwritten by lodash. Check that there are no lodash imports.
```

Comment:
Or maybe not use the comment at all and use a more descriptive test title: `it('should ensure the global variable '_' hold the underscore variable and it is not being overwritten by another library')`?

Reply:
Ok, I've moved the supporting comment into the test description, but I kept the wording actionable to help a future dev know what could be overwriting the variable.